modules/wezterm/hm.nix

```diff
@@ -1,16 +1,21 @@
-{ config, lib, ... }:
 {
-  options.stylix.targets.wezterm.enable =
-    config.lib.stylix.mkEnableTarget "wezterm" true;
-
-  config =
-    with config.lib.stylix.colors.withHashtag;
-    lib.mkIf
-      (
-        config.stylix.enable
-        && config.stylix.targets.wezterm.enable
-        && config.programs.wezterm.enable
-      )
+  mkTarget,
+  config,
+  lib,
+  ...
+}:
+mkTarget {
+  name = "wezterm";
+  humanName = "WezTerm";
```

Comment:
Technically, renaming `"wezterm"` to `"WezTerm"` is out of the scope of this PR. But this is fine.